doc/bitcoin-conf.md

```diff
@@ -0,0 +1,11 @@
+# `bitcoin.conf` - Bitcoin configuration file
+
+All command-line options (except for `-conf`) may be specified in a configuration file, and all configuration file options (except for `-includeconf`) may also be specified on the command line. Command-line options override values set in the configuration file and configuration file options override values set in the GUI.
+
+The configuration file is a plain text file and consist of 'setting=value' entries, one per line. Leading and trailing whitespaces are removed. A number sign (`#`) indicates a comment. Comments and empty lines are ignored.
```

Comment:
Use back-ticks also for `setting=value` to make it consistent?

Reply:
maybe "(`#`) indicates the start a comment, and will make the parser ignore until the end of the line"